### PR TITLE
ZK-4828: remove externally loaded resources with internal refs.

### PR DESCRIPTION
**Pull request by b952426**
ZK-4828: remove externally loaded resources with internal refs.

### CODE REVIEW

File: zk/src/archive/web/WEB-INF/tld/zk/core.dsp.tld [hunk 1]
```diff
@@ -79,6 +79,7 @@ Copyright (C) 2005 Potix Corporation. All Rights Reserved.
 		</function-signature>
 		<description>
 	Returns the edition whether valid or invalid.
+	Deprecated sicne 9.6.0
```
Comment (scribetw) [inline zk/src/archive/web/WEB-INF/tld/zk/core.dsp.tld:82]: since

File: zk/src/org/zkoss/zk/fn/ZkFns.java [hunk 2]
```diff
@@ -81,6 +81,7 @@ public static String getEdition() {
 	/**
 	 * Returns the edition whether valid or invalid.
 	 * @since 6.5.5
+	 * @deprecated since 9.6.0
 	 */
 	public static boolean isEditionValid() {
```
Comment (scribetw) [inline zk/src/org/zkoss/zk/fn/ZkFns.java:86]: `@Deprecated`

File: zkdoc/release-note [hunk 3]
```diff
@@ -38,11 +38,13 @@ ZK 9.6.0
   ZK-4839: za11y: hover navitem without children (JS Error)
   ZK-4833: CKEditor fails to initialized if added to a closed detail component
   ZK-4845: notified @load binding not updating inside shadow element
+  ZK-4828: remove externally loaded resources with internal refs.
 
 * Upgrade Notes
   + The default desktop ID generator was replaced by a more secured one.
     If you prefer to use the previous generator, use org.zkoss.zk.ui.impl.LegacyDesktopIdGenerator.
   + Due to ZK-4783, the hbox of message box is substituted by DIV.
+  + Deprecated the method of isEditionValid() and encodeWithZK() of org.zkoss.zk.fn.ZkFns.
```
Comment (scribetw) [inline zkdoc/release-note:47]: and in DSP

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by plijyfes — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**